proxy/mux.go

```diff
@@ -143,7 +143,11 @@ func (mux *TCPMux) muxConnection(conn net.Conn) {
 
 	// Read in the mux header (a 6-byte representation of a TCP address)
 	muxHeader := make([]byte, 6)
-	if _, err := conn.Read(muxHeader); err != nil {
+	if _, err := conn.Read(muxHeader); err == io.EOF {
+		log.Debug("Detected closed connection in mux.")
```

Comment:
With a busy system with many clients, the log will get a continuous stream of:
WARN[0090] Unable to read valid mux header. Closing connection  address=[::]:22250 error=EOF location=mux.go:157 logger=proxy remoteaddr=x.x.x.x:xxxx